src/app/dataset-view/additional-components/dataset-settings-component/dataset-settings.component.ts

```diff
@@ -86,7 +110,15 @@ export class DatasetSettingsComponent extends BaseComponent implements OnInit {
     }
 
     public ngOnInit(): void {
-        this.activeTab = this.getSectionFromUrl() ?? SettingsTabsEnum.GENERAL;
+        this.initializationActiveTab();
+
+        this.router.events
+            .pipe(filter((event) => event instanceof NavigationEnd))
+            .pipe(takeUntilDestroyed(this.destroyRef))
+            .subscribe(() => {
+                this.activeTab = this.getSectionFromUrl() ?? SettingsTabsEnum.GENERAL;
+                this.cdr.detectChanges();
+            });
```

Comment:
Can the active tab be an input port as well?

Reply:
Need refactored with separate route